constants.hpp

```diff
@@ -22,6 +22,8 @@ along with CUDAProb3++.  If not, see <http://www.gnu.org/licenses/>.
 
 namespace cudaprob3{
 
+    FLOAT_T EarthRadius = 6371.0;
```

Comment:
This seems to break CUDA enabled builds, but I _think_ all that's needed is an extra `HOSTDEVICEQUALIFIER` here, which gets things to compile anyway, but I'll let @dbarrow257 confirm

```suggestion
    HOSTDEVICEQUALIFIER
    FLOAT_T EarthRadius = 6371.0;
```